sbin/md5/md5.c

```diff
@@ -313,7 +319,11 @@ gnu_check(const char *checksumsfile)
 
 	if (strcmp(checksumsfile, "-") == 0)
 		inp = stdin;
+#ifdef HAVE_CAPSICUM
+	else if ((inp = fileargs_fopen(fa, checksumsfile, "r")) == NULL)
```

Comment:
I think there's no need to have HAVE_CAPSICUM anymore. If WITH_CASPER is not defined, the fileargs_* functions will automatically provide non-casper implementations. See cap_filesargs.h.

Reply:
Fixed.

Reply:
> I think there's no need to have HAVE_CAPSICUM anymore. If WITH_CASPER is not defined, the fileargs_* functions will automatically provide non-casper implementations. See cap_filesargs.h.

I think we need HAVE_CAPSICUM because otherwise /usr/src/rescue fails to compile:

```
ld: error: undefined symbol: fileargs_init
>>> referenced by _$$hide$$ md5.lo md5.c
>>>               md5.lo:(_$$hide$$ md5.lo main)

ld: error: undefined symbol: fileargs_fopen
>>> referenced by _$$hide$$ md5.lo md5.c
>>>               md5.lo:(_$$hide$$ md5.lo main)

ld: error: undefined symbol: fileargs_free
>>> referenced by _$$hide$$ md5.lo md5.c
>>>               md5.lo:(_$$hide$$ md5.lo main)
cc: error: linker command failed with exit code 1 (use -v to see invocation)
*** Error code 1
```

Reply:
Fixed.